icons/fediverse.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Fediverse</title><path d="M15.42 2.42a2.295 2.295 0 0 1-2.415 2.168 2.295 2.295 0 0 1-2.168-2.417A2.295 2.295 0 0 1 13.254.003a2.295 2.295 0 0 1 2.167 2.416Zm8.572 8.6a2.295 2.295 0 0 1-2.415 2.169 2.295 2.295 0 0 1-2.169-2.417 2.295 2.295 0 0 1 2.417-2.17 2.295 2.295 0 0 1 2.167 2.418zM18.46 21.83a2.295 2.295 0 0 1-2.416 2.166 2.295 2.295 0 0 1-2.167-2.416 2.295 2.295 0 0 1 2.415-2.169 2.295 2.295 0 0 1 2.168 2.417zM6.47 19.905a2.295 2.295 0 0 1-2.416 2.168 2.295 2.295 0 0 1-2.168-2.416 2.295 2.295 0 0 1 2.417-2.168 2.295 2.295 0 0 1 2.167 2.416zM4.59 7.912a2.295 2.295 0 0 1-2.417 2.169A2.295 2.295 0 0 1 .007 7.663a2.295 2.295 0 0 1 2.415-2.167A2.295 2.295 0 0 1 4.59 7.91Zm-.065 1.014A2.525 2.525 0 0 1 3.446 10l5.929 5.951 1.428-.725Zm7.82 7.85-1.43.724 3.004 3.016A2.525 2.525 0 0 1 15 19.441zm6.86-5.494-3.358 1.7.248 1.583 3.797-1.925a2.525 2.525 0 0 1-.69-1.358zM13.9 13.97 5.964 17.99a2.525 2.525 0 0 1 .689 1.36l7.494-3.799Zm-2.554-9.907L7.52 11.538l1.13 1.135 4.054-7.914a2.525 2.525 0 0 1-1.355-.696Zm-4.824 9.42-1.94 3.785a2.525 2.525 0 0 1 1.356.696l1.714-3.347zm-3.11-3.466a2.525 2.525 0 0 1-1.261.263 2.525 2.525 0 0 1-.243-.024l1.13 7.244a2.525 2.525 0 0 1 1.263-.264 2.525 2.525 0 0 1 .243.025zm3.247 9.37a2.525 2.525 0 0 1 .027.508 2.525 2.525 0 0 1-.266.996l7.243 1.162a2.525 2.525 0 0 1-.026-.509 2.525 2.525 0 0 1 .266-.996zm13.26-6.72-3.344 6.526a2.525 2.525 0 0 1 1.356.696l3.344-6.526a2.525 2.525 0 0 1-1.356-.696Zm-4.56-9.231a2.525 2.525 0 0 1-1.08 1.074l5.178 5.198a2.525 2.525 0 0 1 1.08-1.074Zm-4.725-.76L4.09 5.995a2.525 2.525 0 0 1 .69 1.359l6.545-3.317a2.525 2.525 0 0 1-.69-1.36Zm3.634 1.84a2.525 2.525 0 0 1-1.282.274 2.525 2.525 0 0 1-.222-.022l.58 3.712 1.58.253zm-.587 6.122 1.37 8.776a2.525 2.525 0 0 1 1.243-.253 2.525 2.525 0 0 1 .264.03l-1.296-8.3ZM4.78 7.38a2.525 2.525 0 0 1 .029.517 2.525 2.525 0 0 1-.264.99l3.716.597.73-1.427Zm6.368 1.024-.73 1.426 8.777 1.41a2.525 2.525 0 0 1-.025-.504 2.525 2.525 0 0 1 .27-1z"/></svg>
```

Comment:
There is a [slight difference when comparing with the original](https://github.com/simple-icons/simple-icons/assets/35000807/457a9682-c665-4ccf-9e5a-2914d8274e2f) here: https://gitlab.com/ayush-sharma/to-the-fediverse/-/blob/c046ba7f3ea4c100d9c7538bb42593c245fdf532/static/images/icon.svg

Reply:
Again, that is not the official link. Furthermore, I was advised to optimize this with SVGO.